Workbooks/AlsidIoA.json

```diff
@@ -0,0 +1,389 @@
+              "timeContext": {
+                "durationMs": 7776000000
+              },
+              "timeContextFromParameter": "TimeRange",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "graph",
+              "graphSettings": {
+                "type": 2,
+                "topContent": {
+                  "columnMatch": "Severity",
+                  "formatter": 1
+                },
+                "centerContent": {
+                  "columnMatch": "AlertCount",
+                  "formatter": 1,
+                  "numberFormat": {
+                    "unit": 17,
+                    "options": {
+                      "maximumSignificantDigits": 3,
+                      "maximumFractionDigits": 2
+                    }
+                  }
+                },
+                "nodeIdField": "Severity",
+                "graphOrientation": 3,
+                "showOrientationToggles": false,
+                "nodeSize": null,
+                "staticNodeSize": 100,
+                "colorSettings": {
+                  "nodeColorField": "Severity",
+                  "type": 1,
+                  "colorPalette": "default"
+                },
+                "hivesMargin": 5
+              }
+            },
+            "customWidth": "25",
+            "name": "query - 1",
+            "styleSettings": {
+              "maxWidth": "50"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "let threshold = 1;\r\nlet SeverityTable=datatable(Severity:string,Level:int) [\r\n\"low\", 1,\r\n\"medium\", 2,\r\n\"high\", 3,\r\n\"critical\", 4\r\n];\r\nafad_parser\r\n| where MessageType == 2 \r\n| lookup kind=leftouter SeverityTable on Severity\r\n| where Level >= ['threshold']\r\n| summarize Count = count() by bin(Time, {SamplingPeriod:seconds}), Severity",
+              "size": 0,
+              "title": "Alerts raised over time grouped by severity",
+              "noDataMessage": "No alerts",
+              "noDataMessageStyle": 3,
+              "timeContext": {
+                "durationMs": 7776000000
+              },
+              "timeContextFromParameter": "TimeRange",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "timechart",
+              "chartSettings": {
+                "showLegend": true
+              }
+            },
+            "customWidth": "75",
+            "name": "query - 2"
+          }
+        ]
+      },
+      "name": "Severity",
+      "styleSettings": {
+        "showBorder": true
+      }
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/77941bc6-ccba-4ab4-98b2-296d84ffbc58/resourcegroups/integrator-siem-playground/providers/microsoft.operationalinsights/workspaces/azure-sentinel-francois"
```

Comment:
Workbook should not refer to static resources, Please remove the static values.

Reply:
I just removed "fallbackResourceIds" from the workbook